packages/core/src/lib/toast/toast.svelte

```diff
@@ -0,0 +1,118 @@
+<svelte:options immutable />
+
+<script lang="ts">
+import { type IconName, Icon, Button } from '$lib';
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { ToastVariant, type ToastVariantType } from './variants';
+
+/** The message displayed by the toast */
+export let message: string;
+
+/** The type of toast banner to be displayed to the user */
+export let variant: ToastVariantType;
+
+/** Whether there is an action button displayed */
+export let actionable = false;
+
+/** Whether there is a close icon displayed */
+export let closeable = false;
+
+/** The scaling applied on the x axis of the page*/
+export let progress = 1;
+
+/** Additional CSS classes to pass to the button. */
+let extraClasses: cx.Argument = '';
+export { extraClasses as cx };
+
+const dispatch = createEventDispatcher<{
+  /** when the toast close button is pressed*/
+  close: null;
+}>();
+
+const handleClose = () => dispatch('close');
+
+// $: isUploadingFile = variant === ToastVariant.UploadingFile;
```

Comment:
✂️

Reply:
Removing this file from the PR as it was added in error.